### PR TITLE
Fix potat yes and also add Decoration category

### PR DESCRIPTION
**Pull request by sylv256**

### CODE REVIEW

File: src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java [hunk 1]
```diff
@@ -17,7 +17,6 @@
 
     public TaterModule() {
         super(Proton.identifier("tater"));
-        this.enabled = false;
```
Comment (namebee) [inline src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java:20]: This still shouldn't be enabled by default, Potato's are not the base of the mod.

**Review by namebee — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java (hunk 1 above)
Comment (dzwdz) [reply src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java:20]: ok so how would you disable it

Comment (namebee) [reply src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java:20]: Maybe we should ask the creator of tiny-config.

Comment (sylv256) [reply src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java:20]: Until then, I'll re-disable it for now.

**Comment by sylv256** [comment]
i'm stupid so i put this on the wrong branch

**Pull request closed without merging**